.github/workflows/rabbitmq.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rabbitmq-version: ['3.6', '3.7', '3.8']
+        rabbitmq-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
Not saying we should test all these versions, but wanted to run them at least once to make sure they all work.